nextjs-metrics-viewer/app/(protected)/upload/Components/Modal/EditData.tsx

```
@@ -0,0 +1,83 @@
+import { Button, Modal } from "flowbite-react";
+import { Formik, Field, Form, FormikHelpers } from "formik";
+interface props {
+  initialData: Record<string, string>;
+  open: boolean;
+  onClose: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)